user_guide_src/source/changelogs/v4.1.5.rst

```diff
@@ -14,6 +14,8 @@ BREAKING
 
 - Fixed `a bug <https://github.com/codeigniter4/CodeIgniter4/issues/2913>`_ on CSRF protection. Now CSRF protection works on PUT/PATCH/DELETE requests when CSRF filter is applied. If you use such requests, you need to send CSRF token.
 - In the previous version, if you didn't provide your own headers, ``CURLRequest`` would send the request-headers from the browser, due to a bug. As of this version, it does not send them.
+- Fixed ``BaseBuilder::insertBatch()`` return value. Now it returns SQL string array instead of wrong affected row number when testMode.
+- Because of the optimization, when ``BaseBuilder::insertBatch()`` and ``BaseBuilder::updateBatch()`` are used, the return value of ``$query->getOriginalQuery()`` changed.
```

Comment:
```suggestion
- Major optimizations have been made to the way data is processed in ``BaseBuilder::insertBatch()`` and ``BaseBuilder::updateBatch()`` methods. This resulted in reduced memory usage and faster query processing. As a trade-off, the result generated by the ``$query->getOriginalQuery()`` method was changed. It no longer returns the query with the binded parameters, but the actual query that was run.
```
I think we should be a little more specific in terms of explaining the changes that we've made and that could potentially affect developers.

Reply:
Thanks! My English skill limits the explanation, and I'm not sure how detailed we write to changelog.

And I believe there are few developers to use `$query->getOriginalQuery()` for insertBatch/updateBatch.
So I didn't think it was necessary to go into too much detail.